core/services/log/helpers_test.go

```diff
@@ -95,9 +94,9 @@ func (helper *broadcasterHelper) start() {
 }
 
 func (helper *broadcasterHelper) startWithLatestHeadInDb(head *models.Head) {
+	helper.lb.SetLatestHeadFromStorage(head)
```

Comment:
Is it possible to pass in the head when we instantiate the log broadcaster?

Reply:
hmm we could do for the tests, but for the code in application.go - not at the moment, no.
We get the head from HT, which is not created before LB is created (as LB is passed when we instantiate HT). But this might be possible in the other PR about HeadBroadcaster